pkg/helmpath/lazypath_unix.go

```diff
@@ -43,3 +44,10 @@ func configHome() string {
 func cacheHome() string {
 	return filepath.Join(homedir.HomeDir(), ".cache")
 }
+
+// dataDirs defines the directories relevant directories in use separated with a colon
+//
+// $XDG_DATA_DIRS is usually set by the system
+func dataDirs() string {
```

Comment:
This does not seem to be what the spec suggests doing.

The spec seems to suggest that the system can provide additional paths on XDG_DATA_DIRS that application code (like Helm) may/should consider as other locations where data may be stored. But cache and config are not also to be considered data dirs. This would imply that stuff in the cache was to be treated as data dirs (and likewise with config). I don't think that is what the spec intends.